proof/drefine/Arch_DR.thy

```diff
@@ -761,13 +761,13 @@ next
                           get_index_def transform_cap_list_def
                           dc_def[symmetric]
                    split: cap.split arch_cap.split option.split)
-              (* 45 subgoals, most are irrelevant invocations *)
+              (* 46 subgoals, most are irrelevant invocations *)
```

Comment:
Thanks for updating the comment, these ugly proofs are a pain to update even with them, they're a nightmare without or if they get out of date.